packages/kit/src/views/Home/components/NFTListView/index.tsx

```
@@ -157,9 +157,16 @@ function NFTListView(props: IProps) {
     },
   );
 
-  if (!initialized && isLoading) {
-    return <NFTListLoadingView />;
-  }
+
+  const EmptyComponentElement = useMemo(() => {
```

ESLint

[error] Delete `⏎`

(prettier/prettier)